translations/texts/objects/apex/apexcoolchair/apexcoolchair.object.json

```
@@ -7,7 +7,8 @@
       ]
     },
     "Texts": {
-      "Eng": "An egg shaped chair. It's pretty cool I guess."
+      "Eng": "An egg shaped chair. It's pretty cool I guess.",
+      "Rus": "Кресло в форме яйца. Это довольно круто, я думаю."
```

```suggestion
      "Rus": "Кресло в форме яйца. Весьма круто, я считаю."
```
Поправил стилистику новакидов